utils/ort/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -172,5 +172,10 @@ data class ProcessedDeclaredLicense(
      * The list of all mapped and unmapped licenses.
      */
     @JsonIgnore
-    val allLicenses = spdxExpression?.decompose().orEmpty().map { it.toString() } + unmapped
+    val allLicenses = decompose().map { it.toString() } + unmapped
+
+    /**
+     * Decompose the SPDX expression or return an empty string.
```

Comment:
`[Decompose][SpdxExpression.decompose] the [spdxExpression] or return an empty string if it is null.`